nebula.config.js

```diff
@@ -18,12 +12,27 @@ const renderConfigs = fs.readdirSync('test/rendering/data').map((file) => ({
   },
 }));
 
+const { name, version } = require(path.resolve(__dirname, './package.json')); // eslint-disable-line
+
+const versionHash = crypto
+  .createHash('md5')
+  .update(`${name}@${version}`) // TODO - consider using the commit sha since all charts from the same commit could have the same style
```

Comment:
remove the comment (it is only relevant in a mono repo with multiple charts)

Reply:
Done